### ISSUE TITLE
Error "Plugin class loader for connector was not found" during the backup

### ISSUE THREAD

**Issue by akamensky** [itadventurer/kafka-backup#61]
I see multiple error log entries during the start of the process:

```
[2020-04-17 16:56:14,068] ERROR Plugin class loader for connector: 'de.azapps.kafkabackup.sink.BackupSinkConnector' was not found. Returning: org.apache.kafka.connect.runtime.isolation.DelegatingClassLoader@5b068087 (org.apache.kafka.connect.runtime.isolation.DelegatingClassLoader:165)
```

**Comment by itadventurer** [comment]
The jar file is not built 🤔 hmm that's obvious: I am using a simlink to the jar file and check in the bash script whether the file exists. Need to check whether the link Target exists. Works in docker but not here. Ooops 🙈

**Comment by akamensky** [comment]
It would be good if it was built into the Jar file that is produced on compilation...

**Comment by itadventurer** [comment]
Fixed with #73: Please use the release builds (coming very very soon)
